models/issue_mail.go

```diff
@@ -87,7 +88,9 @@ func mailIssueCommentToParticipants(e Engine, issue *Issue, doer *User, content
 		names = append(names, participants[i].Name)
 	}
 
-	SendIssueCommentMail(issue, doer, content, comment, tos)
+	for _, to := range tos {
+		SendIssueCommentMail(issue, doer, content, comment, []string{to})
```

Comment:
Shouldn't we send emails to all users which have there email adresse disclosed at once?

Reply:
That's what I suggested in the related issue. But I dont't know if it's possilble to do.

Reply:
yes, that is possible, however I wanted to resolve the security issue before adding any enhancements.